src/v2/Apps/Settings/Routes/Saves/Components/SettingsSavesProfiles.tsx

```diff
@@ -0,0 +1,217 @@
+                            key={internalID}
+                            partner={profile.owner}
+                          />
+                        )
+
+                      case "Fair":
+                        return (
+                          <FairEntityHeaderFragmentContainer
+                            key={internalID}
+                            fair={profile.owner}
+                          />
+                        )
+
+                      case "FairOrganizer":
+                        return (
+                          <FairOrganizerEntityHeaderFragmentContainer
+                            key={internalID}
+                            fairOrganizer={profile.owner}
+                          />
+                        )
+                    }
+                  })()}
+                </Column>
+              )
+            })}
+          </GridColumns>
+
+          {relay.hasMore() && (
+            <Box textAlign="center" mt={4}>
+              <Button onClick={handleClick} loading={loading}>
+                Show More
+              </Button>
+            </Box>
+          )}
+        </>
+      ) : (
+        <Text variant="lg" color="black60">
+          Nothing yet.
+        </Text>
+      )}
+    </>
+  )
+}
+
+export const SETTINGS_SAVES_PROFILES_QUERY = graphql`
+  query SettingsSavesProfilesQuery($after: String) {
+    me {
+      ...SettingsSavesProfiles_me @arguments(after: $after)
+    }
+  }
+`
+
+export const SettingsSavesProfilesPaginationContainer = createPaginationContainer(
+  SettingsSavesProfiles,
+  {
+    me: graphql`
+      fragment SettingsSavesProfiles_me on Me
+        @argumentDefinitions(after: { type: "String" }) {
+        followsAndSaves {
+          profilesConnection(first: 12, after: $after)
+            @connection(key: "SettingsSavesProfiles_profilesConnection") {
+            totalCount
+            edges {
+              node {
+                internalID
+                profile {
+                  name
+                  href
+                  avatar: image {
+                    cropped(width: 45, height: 45) {
+                      src
+                      srcSet
+                    }
+                  }
+                  owner {
+                    __typename
+                    ... on Partner {
+                      ...PartnerEntityHeader_partner
+                    }
+                    ... on Fair {
+                      ...FairEntityHeader_fair
+                    }
+                    ... on FairOrganizer {
+                      ...FairOrganizerEntityHeader_fairOrganizer
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    `,
+  },
+  {
+    direction: "forward",
+    getFragmentVariables(prevVars, totalCount) {
+      return { ...prevVars, totalCount }
+    },
+    getVariables(_, { cursor: after }, fragmentVariables) {
+      return { ...fragmentVariables, after }
+    },
```

Comment:
assuming all of these args are needed? (from [this post](https://artsy.github.io/blog/2020/08/31/relay-pagination-containers-demystified/))

Reply:
Yep, all required actually.